mips/clock.c

```diff
@@ -7,20 +7,19 @@
 #include <sched.h>
 #include <sysinit.h>
 
-/* This counter is incremented every millisecond. */
-static volatile realtime_t mips_clock_ms;
+/* System clock gets incremented every milisecond. */
+static timeval_t *sys_clock = &TIMEVAL_INIT(0, 0);
```

Comment:
This is a part of C99 standard and is called *designated initializers*. It's not a *temporary*, but an *anonymous* value. On the top level it gets allocated in `.data` section, inside function it gets allocated on the stack.